assets/css/design.css

```diff
@@ -0,0 +1,220 @@
+.image-replacement {
+  text-indent: -1000%;
+  white-space: nowrap;
+  overflow: hidden; }
+
+.container {
+  max-width: 63.333333333em;
+  padding: 0 1em;
+  margin: 0 auto;
+  position: relative; }
+
+.unstyled-list, .unstyled {
+  margin-left: 0;
+  padding-left: 0;
+  list-style: none outside none; }
+
+.inline-list {
+  margin-left: -0.5em;
+  margin-bottom: 0; }
+  .inline-list li {
+    display: inline-block;
+    margin-left: 0.5em; }
+
+.text--center {
+  text-align: center; }
+
+.bg-default {
+  background: lightblue; }
+
+/* Welcome to design page */
+body {
+  background-color: #f3f1eb; }
+
+.design-hero {
+  padding: 3em 0 6em;
+  background-image: url("../img/design-hero-bg.jpg");
+  background-position: center top;
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: #fff; }
+  @media (min-width: 40em) {
+    .design-hero {
+      padding: 6em 0 9em;
+      background-position: center top; } }
+  @media (min-width: 80em) {
+    .design-hero {
+      background-position: center -100px; } }
+  .design-hero h1 {
+    font-weight: 600;
+    font-size: 3.5em; }
+    @media (min-width: 40em) {
+      .design-hero h1 {
+        font-size: 5.5em; } }
+  .design-hero .subhead {
+    margin-bottom: 0;
+    font-size: 1.333333333em; }
+
+.page {
+  background-color: #fff;
+  padding: 3em 0;
+  position: relative;
+  top: -4em; }
+  @media (min-width: 40em) {
+    .page {
+      padding: 6em 0;
+      top: -8em; } }
+  .page > p {
+    max-width: 38em;
+    margin-left: auto;
+    margin-right: auto; }
+
+.intro-section {
+  padding: 0 0 4em 0;
+  border-bottom: 1px solid #e4e3dd; }
+  .intro-section > p {
+    font-size: 1.166666667em;
+    padding: 0 1em 0 1em; }
+    @media (min-width: 40em) {
+      .intro-section > p {
+        padding: 0 5em 0 4em; } }
+    @media (min-width: 60em) {
+      .intro-section > p {
+        padding-right: 15em;
+        padding-left: 8em; } }
+  .intro-section h1 {
+    font-size: 2em;
+    font-weight: 700;
+    max-width: 14em;
+    padding-left: 0.5em;
+    padding-right: 0.5em;
+    background: linear-gradient(to right, #61b252, #54b1e4);
```

Comment:
Am I right in thinking, on non-webkit browsers, this will render as a big rectangle of gradient, with black text on top?

In which case, shouldn't you maybe use `-webkit-linear-gradient` to make sure that only webkit browsers get the gradient background?